src/java.base/share/classes/java/util/PropertyPermission.java

```diff
@@ -334,16 +334,12 @@ private static int getMask(String actions) {
      * @return the canonical string representation of the actions.
      */
     static String getActions(int mask) {
-        switch (mask & (READ|WRITE)) {
-            case READ:
-                return SecurityConstants.PROPERTY_READ_ACTION;
-            case WRITE:
-                return SecurityConstants.PROPERTY_WRITE_ACTION;
-            case READ|WRITE:
-                return SecurityConstants.PROPERTY_RW_ACTION;
-            default:
-                return "";
-        }
+        return switch (mask & (READ | WRITE)) {
```

Comment:
Just a suggestion, but it might be more readable if you align the lambda operators

Reply:
Thank you for pointing to this. Actually, I just noticed that there was some kind of vertical alignment before my change. I added vertical alignment for single-line switch rules.